pyglet/window/cocoa/pyglet_view.py

```diff
@@ -117,7 +118,6 @@ def initWithFrame_cocoaWindow_(self, frame, window):
         # "Option-e", "e" if the protocol isn't implemented.  So the easiest
         # thing to do is to subclass NSTextView which *does* implement the
         # protocol and let it handle text input.
-        PygletTextView = cocoapy.ObjCClass('PygletTextView')
```

Comment:
I'm not entirely sure if this is correct or not. Originally PygletTextView was being imported in pyglet/window/cocoa/__init.py__. Let me know if there is a better way to handle this

Reply:
I don't see an issue with this as long as it works.
It doesn't actually seem to be used in `__init__.py` after the import, so I'm not sure why it was being imported there.

Reply:
This should be good to go then